crates/uv-resolver/src/resolver/mod.rs

```diff
@@ -1719,14 +1722,7 @@ impl SolveState {
             }
 
             if let Some(for_package) = for_package {
-                if self.markers == MarkerTree::And(Vec::new()) {
-                    debug!("Adding transitive dependency for {for_package}: {package}{version}",);
-                } else {
-                    debug!(
-                        "Adding transitive dependency for {for_package}{{{}}}: {package}{version}",
-                        self.markers
-                    );
-                }
+                debug!("Adding transitive dependency for {for_package}: {package}{version}");
```

Comment:
Undoing this change since we don't need it on every package, we only need it once per fork.